tests/test_snql_api.py

```diff
@@ -580,6 +580,24 @@ def test_suspect_spans_data(self) -> None:
         assert len(data) == 1
         assert data[0]["array_spans_exclusive_time"] > 0
 
+    def test_attribution_tags(self) -> None:
+        response = self.post(
+            "/events/snql",
+            data=json.dumps(
+                {
+                    "query": f"""MATCH (events)
+                    SELECT count() AS count
+                    WHERE timestamp >= toDateTime('{self.base_time.isoformat()}')
+                    AND timestamp < toDateTime('{self.next_time.isoformat()}')
+                    AND project_id IN tuple({self.project_id})
+                    """,
+                    "team": "sns",
+                    "feature": "test",
+                }
+            ),
+        )
+        assert response.status_code == 200
```

Comment:
Is there any way for us to test the metrics we output? I'm not saying you should do it in this PR if we don't but I think we probably should?

Reply:
I checked and right now we don't. I don't think it would be hard to add/expand the `DummyMetrics` class to facilitate those types of tests, but I didn't want to do that in this PR.